tests/test_portal.py

```diff
@@ -268,16 +272,20 @@ def test_redirect_validation(self):
 
         client = self.add_client()
         client_url = client._redirect_uris
-        local_url = "http://{}/home?test".format(self.app.config.get('SERVER_NAME'))
+        local_url = "http://{}/home?test".format(
+            self.app.config.get('SERVER_NAME'))
         invalid_url = 'http://invalid.org'
 
         # validate redirect of /website-consent-script GET
-        response = self.client.get('/website-consent-script/{}?redirect_url='
-                             '{}'.format(TEST_USER_ID, local_url))
+        response = self.client.get(
+            '/website-consent-script/{}?redirect_url={}'.format(
```

Comment:
Can you refactor these to use the `query_string` keyword argument?
ie `query_string={'redirect_url': local_url}`

Reply:
@ivan-c so in this case would it look like:
````
response = self.client.get(
    '/website-consent-script/{}'.format(TEST_USER_ID),
    query_string={'redirect_url': local_url}
 )
````
?